integration_tests/conftest.py

```diff
@@ -24,8 +24,22 @@ def populations(sim) -> List[pd.DataFrame]:
     population_states = []
     for _ in range(max(TIME_STEPS_TO_TEST) + 1):
         pop = sim.get_population(untracked=True)
-        population_states.append(pop)
+        pipelines = sim.list_values()
+        for pipeline in pipelines:
+            p = sim.get_value(pipeline)(pop.index)
+            # Metrics is a dict we cannot concat and do not want to
+            if isinstance(p, dict):
+                continue
+            elif isinstance(p, pd.DataFrame):
+                renamed_cols = [f"{pipeline}.{col}" for col in p.columns]
+                rename_dict = dict(zip(p.columns, renamed_cols))
+                p = p.rename(columns=rename_dict)
+                pop = pd.concat([pop, p], axis=1)
```

Comment:
```suggestion
                pop = pd.concat([pop, p.add_prefix(f"{pipeline}.")], axis=1)
```

Reply:
@rmudambi Did you and Jim decide that this wouldn't be a good change/wouldn't work? Nbd if so just wanted to make sure!

Reply:
I do not think it matters for setup of tests but I have updated it as it is a lot cleaner.